docs/create-packages/multiple-target-frameworks-project-file.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Multi-targeting for NuGet Packages in your project file
+description: Description of the various methods to target multiple .NET Framework versions from within a single NuGet package.
+author: karann-msft
+ms.author: karann
+ms.date: 07/15/2019
+ms.topic: conceptual
+---
+
+# Support multiple .NET Framework versions in your project file
+
+When you first create a project, we recommend you create a .NET Standard class library, as it provides compatibility with the widest range of consuming projects. By using .NET Standard, you add [cross-platform support](/dotnet/standard/library-guidance/cross-platform-targeting) to a .NET library by default. However, in some scenarios, you may also need to include code that targets a particular framework. This article shows you how to do that for [SDK-style](../resources/check-project-format.md) projects.
+
+For SDK-style projects, you can configure support for multiple targets frameworks ([TFM](/dotnet/standard/frameworks)) in your project file, then use `dotnet pack` or `msbuild /t:pack` to create the package.
+
+> [!NOTE]
+> nuget.exe CLI does not support packing SDK-style projects, so you should only use `dotnet pack` or `msbuild /t:pack`. We recommend that you [include all the properties](../reference/msbuild-targets.md#pack-target) that are usually in the `.nuspec` file in the project file instead. To target multiple .NET Framework versions in a non-SDK-style project, see [Supporting multiple .NET Framework versions](supporting-multiple-target-frameworks.md).
+
+## Create a project that supports multiple .NET Framework versions
+
+1. Create a new .NET Standard class library either in Visual Studio or use `dotnet new classlib`.
+
+   We recommend that you create a .NET Standard class library for best compatibility.
+
+2. Edit the *.csproj* file to support the target frameworks.
+
+   For example, change `<TargetFramework>netstandard2.0</TargetFramework>` to `<TargetFrameworks>netstandard2.0;net45</TargetFrameworks>`.
+
+   Make sure that you change the XML element changed from singular to plural (add the "s" to both the open and close tags).
+
+3. If you have any code that only works in one TFM, you can use `#if NET45` or `#if NETSTANDARD20` to separate TFM-dependent code. For example, you can use the following code:
```

Comment:
Link https://docs.microsoft.com/en-us/dotnet/core/tutorials/libraries#how-to-multitarget maybe?